jujugui/static/gui/src/app/components/deployment-flow/credential/credential.js

```diff
@@ -77,30 +75,57 @@ YUI.add('deployment-credential', function() {
       @method _getCredentials
     */
     _getCredentials: function() {
-      this.credentialXHR = this.props.listTemplates(
-        this._getCredentialsCallback);
+      const cloud = this.props.cloud;
+      const user = this.props.users.jem.user;
+      if (user) {
+        this.props.getTagsForCloudCredentials(
+          [[user, `cloud-${cloud}`]],
```

Comment:
Cloud tags are already provided by the server. Why are we recalculating them from cloud names here? I know that the loic is quite easy and this will probably work forever, but the code would feel more solid if we just used the tags returned from the server.